js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -176,6 +169,10 @@ export const AssetGraphExplorer = (props: Props) => {
         'push',
       );
     }, [explorerPath, onChangeExplorerPath]),
+    assetTags: filters?.tags || emptyArray,
+    setAssetTags: filters?.tags ? setAssetTags : undefined,
+    owners: filters?.owners || emptyArray,
+    setOwners: filters?.owners ? setOwners : undefined,
```

Comment:
This all looks good! It vaguely seems like the `filters / setFilters` should be passed into this hook, and then the exploding into the different getters/setters should happen there (instead of them being broken out and then passed in one at a time here), but maybe they're also required down below line 178...